docs/docs/50-user-guide/70-annotations.md

```diff
@@ -0,0 +1,74 @@
+---
+sidebar_label: Annotations
+description: List of Kargo-specific annotations and labels
+---
+
+# Annotations and Labels used by Kargo
+
+This page documents Kargo-specific annotations, labels, and finalizers. Some of these fields can be set by users, while others are system-managed and used internally by the control plane.
+
+## Annotations
+
+| Annotation key | Target resource(s) | Possible values | Description |
+|----------------|--------------------|------------------|-------------|
+| `kargo.akuity.io/refresh` | Any Kargo resource | Arbitrary string (acts as a token) | Triggers reconciliation when the value changes. Useful for forcing a resync. |
+| `kargo.akuity.io/reverify` | `Stage` | Verification ID or JSON object (VerificationRequest) | Triggers re-verification of the currently assigned Freight. |
+| `kargo.akuity.io/abort` | `Stage` | Verification ID | Aborts an in-progress Freight verification. |
+| `kargo.akuity.io/description` | Any | Free-form string | Human-readable description displayed in the UI. |
+| `kargo.akuity.io/color` | `Stage` | Hex color code (e.g. `#ff8800`) | Optional cosmetic color used in UI. |
+| `kargo.akuity.io/create-actor` | All | Actor identity (username/service) | Injected by the control plane to indicate who created the resource. |
+| `kargo.akuity.io/authorized-stage` | `Freight`, `Promotion`, etc. | `<project>:<stage>` | Indicates which Stage is authorized to manage the resource. |
+| `kargo.akuity.io/promotion` | `PromotionTask`, internal resources | Promotion ID or name | Links the resources to a promotion process. |
```

Comment:
This would neither appear on a `PromotionTask` nor on _most_ resource types.

It doesn't link a resource to a "promotion process." It links a resource to a _Promotion_.

This is also another one that's system-assigned and may be better to omit until there's a solid reason not to.